### PR TITLE
Move addons into builder subdirectory

### PR DESCRIPTION
**Pull request by dhylands**
Since the checksum changes each time, we don't deploy
directly into the root of mozilla-gateway-addons but rather
into a builder subdirectory.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -12,5 +12,5 @@
 	branch = master
 [submodule "zwave-adapter"]
 	path = zwave-adapter
-	url = https://github.com/mozilla-iot/zwave-adapter
-	branch = master
+	url = https://github.com/dhylands/zwave-adapter
```
Comment (mrstegeman) [inline .gitmodules]: Probably should change this back.

Comment (dhylands) [reply .gitmodules]: Will do. I noticed that failures in package.sh weren't stopping the build, so I wanted to test using bash -e in the package.sh which seems to work.

**Comment by dhylands** [comment]
OK - this seems to be working now. I pointed the zwave-adapter back to master.

File: build-adapter.sh [hunk 2]
```diff
@@ -44,17 +48,62 @@ if [ -f "package.json" ]; then
   rm -rf node_modules
 fi
 
-if [ "${ADDON_ARCH}" == "linux-arm" ]; then
```
[the remark this answers is not in the document]
Comment (dhylands) [reply build-adapter.sh]: Ohhh. Good catch.

File: build-adapter.sh [hunk 3]
```diff
@@ -44,17 +48,62 @@ if [ -f "package.json" ]; then
...
+  # Build and install the OpenZWave library.
+  # The hash used here should match the hash from rpi-image-builder
+  OPENZWAVE_HASH="ab5fe966fee882bb9e8d78a91db892a60a1863d9"
+  OPENZWAVE_ZIP="https://codeload.github.com/OpenZWave/open-zwave/zip/${OPENZWAVE_HASH}"
+  rm -rf openzwave.zip open-zwave-${OPENZWAVE_HASH}
+  curl --output openzwave.zip ${OPENZWAVE_ZIP}
+  unzip -q openzwave.zip
+
+  OPEN_ZWAVE="OpenZWave/open-zwave"
+  mkdir -p ${OPEN_ZWAVE}
```
[the remark this answers is not in the document]
Comment (dhylands) [reply build-adapter.sh]: I think I'll simplify things and make OPEN_ZWAVE just be open-zwave then I can remove the mkdir

**Pull request merged**